### ISSUE TITLE
Rule proposal: prefer `||` instead of ternary conditional operator

### ISSUE THREAD

**Issue by jimmywarting** [sindresorhus/eslint-plugin-unicorn#2193]
### Description

I saw some unnecessary code in a project that used this pattern:
```js
process.env.cdn ? process.env.cdn : 'https://cdn.example.com'
```
when the expression and the first value are the same `process.env.cdn ? process.env.cdn`, then you could just as easily write it as a `||` instead. and avoid having to do property lookup on `process` again and again. (and potentially trigger any additional getter fn or what not)

### Fail

```js
process.env.cdn ? process.env.cdn : 'https://cdn.example.com'
```

### Pass

```js
process.env.cdn || 'https://cdn.example.com'
```

**Comment by fisker** [comment]
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-logical-operator-over-ternary.md

**Comment by jimmywarting** [comment]
Ah, thanks!
